src/main/java/TaskList.java

```diff
@@ -0,0 +1,67 @@
+import java.util.ArrayList;
+
+/**
+ * list of tasks
+ */
+public class TaskList {
+    public ArrayList<Task> list;
+
+    public TaskList() {
+        list = new ArrayList<>();
+    }
+
+    /**
+     * add the task to the list
+     * 
+     * @param t task to be added
+     */
+    public void add(Task t) {
+        list.add(t);
+    }
+
+    /**
+     * delete the task from the list
+     * 
+     * @param i index of the task to be deleted
+     * @return returns the task that was deleted
+     */
+    public Task remove(int i) {
+        return list.remove(i-1);
+    }
+
+    /**
+     * fetches the task from the list
+     * 
+     * @param i index of the task mentioned
+     * @return the task with the index i
+     */
+    public Task get(int i) {
+        return list.get(i);
+    }
+    /**
+     * 
```

Comment:
The Javadoc in this section is missing a description of what the function does.

Reply:
I thought the function name, param and return statement made it quite self-explanatory, but I will add that in.